hail/python/hailctl/dataproc/connect.py

```diff
@@ -54,12 +54,25 @@ def main(args, pass_through_args):
         stderr=sp.STDOUT
     )
 
-    for c in ['chromium', 'chromium-browser']:
-        chrome = which(c)
-        if chrome is not None:
-            break
-    if chrome is None:
+    import platform
+    system = platform.system()
+
+    if system == 'Darwin':
         chrome = r'/Applications/Google Chrome.app/Contents/MacOS/Google Chrome'
+        user_data_dir = '/tmp/'
+    elif system == 'Linux':
+        import shutil
+        chrome = None
+        for c in ['chromium', 'chromium-browser']:
+            chrome = chrome or shutil.which(c)
+        if chrome is None:
+            raise EnvironmentError("cannot find 'chromium' or 'chromium-browser' on path")
+        user_data_dir = '/tmp'
+    elif system == 'Windows':
+        chrome = r'/mnt/c/Program Files (x86)/Google/Chrome/Application/chrome.exe'
```

Comment:
Should this be a windows path? This seems like it would only work for cygwin or WSL (I am not overly familiar with these.)

Reply:
This is what Akhil was using, I think. I'm mostly just copying the stuff that Dan had done to set him up into here. I'll try deploying a pypi dev build, installing on my windows machine, and testing

Reply:
I'm pretty cool to merge this. It won't make our windows support worse :stuck_out_tongue_winking_eye: